plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java

```diff
@@ -61,14 +61,14 @@ abstract public class AbstractBash extends Task {
 
     @Builder.Default
     @Schema(
-        title = "Interpreter args used"
+        title = "Interpreter arguments to be used."
```

Comment:
Can someone please look at line 40 of this file, and confirm if that statement is correct?
I see that the default there is `RunnerType.PROCESS` as per line 45: `protected RunnerType runner = RunnerType.PROCESS`.

But then, here I see it is DOCKER: https://github.com/shrutimantri/plugin-scripts/blob/master/plugin-script/src/main/java/io/kestra/plugin/scripts/exec/AbstractExecScript.java#L33

Reply:
default is DOCKER

PROCESS was default very early on but it changed. you can try using any of those and you should see that by default it runs in Docker

Reply:
Is Line 45 correct then?

Reply:
This class is the parent of the old deprectated script tasks that uses the PROCESS runner by default.

I suggest changing the documentation title to just `The script runner` with no mention as default as defaults are automatically documented.